plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/service/internal/orient/OrientMetadataStore.java

```diff
@@ -113,6 +128,46 @@ public void doStop() throws Exception {
   }
 
   /**
+   * Performs DB backup, during which database is "frozen" for data modification attempts.
```

Comment:
During backup all further write operations are locked, see http://www.orientechnologies.com/docs/2.0/orientdb.wiki/Backup-and-Restore.html#when-to-use-backup-and-when-export for more details